### DISCUSSION TITLE
Generating samples without interface or response

### DISCUSSION THREAD

**Discussion by apthorpe** [snl-dakota/dakota#68] [Q&A]
Hi,

Is it possible to run Dakota purely as a sample generator with no interface or response function? For example, running `lhsdrv` against a sample input file from SAND2004-2439 generates samples and describes the correlations between them -- and nothing else. At this point I have no need for the job control and response analysis functions of Dakota; maybe one day, but not today. It seems that I have to create both a null response function and an interface to drive it - is that correct?

**Comment by briadam** [comment]
Dakota binary packages should still include the lhsdrv executable if helpful. If you don’t see it we can try to guide based on the package you’re using.

Select Dakota methods (including LHS and random sampling) support a [pre-run mode](https://snl-dakota.github.io/docs/6.18.0/users/usingdakota/running/executionphases.html) .This will generate a set of sample points to a tabular file and exit. You’ll still need to specify an interface and response block, but the interface can be a dummy script or one of Dakota’s built-in direct drivers such as text_book.

There are other ways to coerce similar behavior; these seem closest to what you’re seeking.
